### PR TITLE
Updated user submitted form with styling and JS changes

### PR DESCRIPTION
**Pull request by TaylorLancaster12**

### CODE REVIEW

File: app/views/user_submitted/pages/_contents_form.erb [hunk 1]
```diff
@@ -70,8 +79,10 @@
             }
           })
           .done(function(data) {
+            $(btn).parent().children('.status-icon').html('<i class="fa fa-check" aria-hidden="true"></i>');
+            $("#prompt").html(data["notice"]).css('display','flex');
             console.log(data, btn);
-            $(btn).parents('.dz-image-preview').hide();
+            // $(btn).parents('.dz-image-preview').hide();
```
Comment (shadoath) [inline app/views/user_submitted/pages/_contents_form.erb]: Looks like we no longer hide the upload box. Can they now update the text?

Comment (strikemike2k) [reply app/views/user_submitted/pages/_contents_form.erb]: Yes they can. They can save the text, modify and save again to overwrite.

Comment (shadoath) [reply app/views/user_submitted/pages/_contents_form.erb]: Most Excellent! Let's remove this comment in that case.

**Review by shadoath — APPROVED** [review]

**Review by strikemike2k — APPROVED** [review]
Great UX improvements.

**Comment by strikemike2k** [comment]
[{NSR} User Submitted form updates](https://trello.com/c/rBbJqnP9/1455-nsr-user-submitted-form-updates)

**Pull request merged**